.github/workflows/validate-workflows.yaml

```diff
@@ -0,0 +1,49 @@
+name: Validate Github Workflows
+
+on:
+  pull_request: {}
+
+jobs:
+  verify-json-validation:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        workflow:
+          - aks.yaml
+          - bpf-checks.yaml
+          - codeql-analysis.yml
+          - cyclonus-netpol-test.yaml
+          - docs.yaml
+          - eks.yaml
+          - gke.yaml
+          - go-check.yaml
+          - images-legacy-base-releases.yaml
+          - images-legacy-hotfix-releases.yaml
+          - images-legacy-releases.yaml
+          - images-legacy.yaml
+          - images.yaml
+          - kind-1.19.yaml
+          - nightly.yaml
+          - smoke-test-ipv6.yaml
+          - smoke-test.yaml
+          - validate-workflows.yaml
```

Comment:
[Not blocking] This is a good starting point but I would probably prefer we get rid of the matrix and directly list files in the `.github/workflows`, and then process them directly. The matrix works fine as-is because we are using GH Actions for processing, but we could do the same with a simple `find -exec` and the actual underlying tools these GH Actions use.

Reply:
Wanted to be really hip with this workflow. Did you see that I actually used a gh action for saving to file? The future is now!

Reply:
Oh this is a good point brought up by @nbusseneau. Having a matrix defined here will mean that we will consume all GH quotas for GH actions immediately as soon this GH action is triggered. Each item in a matrix counts as a GH concurrent action.